app/helpers/quadicon_helper.rb

```diff
@@ -477,7 +467,7 @@ def render_host_quadicon(item, options)
       output << flobj_img_simple('100/shield.png', "g72") unless item.get_policies.empty?
     else
       output << flobj_img_simple
-      output << flobj_img_small(img_for_host_vendor(item), "e72")
+      output << flobj_img_simple(img_for_host_vendor(item), "e72", 64)
```

Comment:
This is a change I made early in the process. It makes the PR too noisy now, I'll just revert these.